README.md

```diff
@@ -1,6 +1,5 @@
-[![Build Status](https://travis-ci.org/elevatorup/solidus_searchkick.svg?branch=master)](https://travis-ci.org/elevatorup/solidus_searchkick)
+[![CircleCI](https://circleci.com/gh/solidusio-contrib/solidus_searchkick.svg?style=svg)](https://circleci.com/gh/solidusio-contrib/solidus_searchkick)
 [![Code Climate](https://codeclimate.com/github/elevatorup/solidus_searchkick/badges/gpa.svg)](https://codeclimate.com/github/elevatorup/solidus_searchkick)
```

Comment:
Does codeclimate use the coverage stuff you removed later?

Reply:
You're right. I'm removing also this right now and I'll open an issue to discuss if and what we want to add as a code analyzer.